<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IvFinderDockWidget</class>
 <widget class="QDockWidget" name="IvFinderDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>168</width>
    <height>217</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>168</width>
    <height>524287</height>
   </size>
  </property>
  <property name="allowedAreas">
   <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
  </property>
  <property name="windowTitle">
   <string>Locate target</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1" colspan="3">
     <widget class="QLineEdit" name="targetNameNonsiderealLineEdit">
      <property name="statusTip">
       <string>Official MPC name of a nonsidereal target</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="3">
     <widget class="QLineEdit" name="targetDeltaLineEdit">
      <property name="statusTip">
       <string>Target declination  (decimal or sexagesimal)</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="3">
     <widget class="QLineEdit" name="targetAlphaLineEdit">
      <property name="statusTip">
       <string>Target right ascension (decimal or sexagesimal)</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Dec</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QToolButton" name="MPCresolverToolButton">
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="statusTip">
       <string>Nonsidereal target: Lookup the coordinates for the DATE-OBS in this image</string>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../dockwidgets/icons/magnifyer.png</normaloff>../../dockwidgets/icons/magnifyer.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>49</width>
        <height>49</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoExclusive">
       <bool>true</bool>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QToolButton" name="targetresolverToolButton">
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="statusTip">
       <string>Sidereal target: Lookup the coordinates</string>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../dockwidgets/icons/magnifyer.png</normaloff>../../dockwidgets/icons/magnifyer.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>49</width>
        <height>49</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoExclusive">
       <bool>true</bool>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="3">
     <widget class="QLineEdit" name="targetNameSiderealLineEdit">
      <property name="statusTip">
       <string>Name of a sidereal target</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="4">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="locatePushButton">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="statusTip">
         <string>Shows manually entered coordinates. The target resolver buttons do this automatically.</string>
        </property>
        <property name="text">
         <string>Locate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearPushButton">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="statusTip">
         <string>Clears all input fields and removes the target marker</string>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>R.A.</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0" colspan="4">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>targetNameSiderealLineEdit</tabstop>
  <tabstop>targetNameNonsiderealLineEdit</tabstop>
  <tabstop>targetAlphaLineEdit</tabstop>
  <tabstop>targetDeltaLineEdit</tabstop>
  <tabstop>locatePushButton</tabstop>
  <tabstop>clearPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
